pkg/administration/events/stream/stream.go

```diff
@@ -3,14 +3,34 @@ package stream
 import (
 	"testing"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/stackrox/rox/pkg/administration/events"
 	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/queue"
+)
+
+const (
+	// Sample calculation with a sample administration event (250 chars in message + hint):
+	// 1 Administration event = 160 bytes
+	// 100000 *160 bytes = 16 MB
+	maxQueueSize = 100000
+)
+
+var (
+	administrationEventsQueueCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Where is this counter registered with Prometheus?

Reply:
Ah, great catch.